br/pkg/storage/local.go

```diff
@@ -73,6 +77,10 @@ func (l *LocalStorage) WalkDir(ctx context.Context, opt *WalkOption, fn func(str
 		// so use Rel to convert to relative path to l.base
 		path, _ = filepath.Rel(l.base, path)
 
+		if !strings.HasPrefix(path, opt.ObjPrefix) {
```

Comment:
maybe `strings.HasPrefix(f.Name(), opt.ObjPrefix)`(which only contains the file name e.g. `foo.log` instead of full path e.g. `/some/dir/foo.log`)?

Reply:
`path` become `filename` after L78.
I had a unit case to cover this.